src/utils/globalNavigasjonUtils.ts

```diff
@@ -105,12 +106,10 @@ const getNumberOfBehandlerBerOmBistandOppgaver = (
 };
 
 const getNumberOfArbeidsuforhetOppgaver = (
-  personoppgaver: PersonOppgave[]
+  arbeidsuforhetVurderinger: VurderingResponseDTO[]
 ): number => {
-  return getAllUbehandledePersonOppgaver(
-    personoppgaver,
-    PersonOppgaveType.ARBEIDSUFORHET_VURDER_AVSLAG
-  ).length;
+  const sisteVurdering = arbeidsuforhetVurderinger[0];
```

Comment:
Kan det være mer enn en?

Reply:
Det kan være flere `arbeidsuforhetVurderinger` for en person, men bare et "aktivt" forhåndsvarsel til enhver tid. Dvs alltid en oppfylt eller avslag-vurdering før et nytt forhåndsvarsel.

Reply:
Kan det kræsje her hvis lista er tom?